package.json

```diff
@@ -117,4 +117,4 @@
   "scripts": {
     "dev-server": "NODE_OPTIONS=--max_old_space_size=4096 npx webpack serve --mode=development"
   }
-}
+}
```

Comment:
nit: Looks like we lost a newline at the end of this file, perhaps an editor issue!

Reply:
I'll resolve this and the merge conflict this morning!